error when building drake as a ROS catkin package

I followed the instruction on http://drake.mit.edu/from_source_ros.html#build-from-source-using-ros-catkin to build drake using catkin. But I get the following message after I call
```
catkin build -DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo
```

```
Errors     << drake_cars_examples:make /home/hongkai/dev/drake_catkin_workspace/logs/drake_cars_examples/build.make.001.log
In file included from /home/hongkai/dev/drake_catkin_workspace/src/drake_ros_integration/drake_cars_examples/src/car_sim_lcm_and_ros.cc:10:0:
/home/hongkai/dev/drake_catkin_workspace/src/drake_ros_integration/drake_ros_systems/include/drake_ros/ros_vehicle_system.h:9:50: fatal error: ackermann_msgs/AckermannDriveStamped.h: No such file or directory
#include "ackermann_msgs/AckermannDriveStamped.h"
                                                 ^
compilation terminated.
make[2]: *** [CMakeFiles/car_sim_lcm_and_ros.dir/src/car_sim_lcm_and_ros.cc.o] Error 1
make[1]: *** [CMakeFiles/car_sim_lcm_and_ros.dir/all] Error 2
make: *** [all] Error 2
cd /home/hongkai/dev/drake_catkin_workspace/build/drake_cars_examples; catkin build --get-env drake_cars_examples | catkin env -si  /usr/bin/make --jobserver-fds=6,7 -j; cd -
.........................................................................................................................................
Failed     << drake_cars_examples:make           [ Exited with code 2 ]
Failed    <<< drake_cars_examples                [ 4.7 seconds ]
[build] Summary: 2 of 3 packages succeeded.
[build]   Ignored:   1 packages were skipped or are blacklisted.
[build]   Warnings:  1 packages succeeded with warnings.
[build]   Abandoned: None.
[build]   Failed:    1 packages failed.
[build] Runtime: 1 minute and 27.9 seconds total.
```

--- a/drake/doc/from_source_ros.rst
+++ b/drake/doc/from_source_ros.rst
@@ -36,6 +36,11 @@ Install
 `ROS Indigo <http://wiki.ros.org/indigo>`_. Other versions of the OS and ROS
 may or may not work.
 
+Install package ``ros-indigo-ackermann-msgs``, which is used by
+software in `drake_ros_systems <https://github.com/RobotLocomotion/drake/tree/master/ros/drake_ros_systems>`_::
+
+    sudo apt-get install ros-indigo-ackermann-msgs
+
 Once the OS and ROS are installed, install
 `Catkin Tools <http://catkin-tools.readthedocs.io/en/latest/>`_ by following
 the instructions
@@ -168,10 +173,6 @@ To run Drake's ROS-powered cars example, first add the
     git clone git@github.com:liangfok/ackermann-drive-teleop.git ackermann_drive_teleop
     cd ackermann_drive_teleop
     git checkout feature/ackermann_drive_stamped
-
-You will also need to install the package ``ros-indigo-ackermann-msgs``::
-
-    sudo apt-get install ros-indigo-ackermann-msgs
 
 Since a new package was added to the ROS workspace, re-build the workspace
 (note that a build type of ``RelWithDebInfo`` is selected since the simulation

--- a/ros/drake_ros_systems/package.xml
+++ b/ros/drake_ros_systems/package.xml
@@ -40,12 +40,15 @@
   <!-- Use test_depend for packages you need only for testing: -->
   <!--   <test_depend>gtest</test_depend> -->
   <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>ackermann_msgs</build_depend>
+  <build_depend>drake</build_depend>
   <build_depend>roscpp</build_depend>
   <build_depend>tf</build_depend>
-  <build_depend>drake</build_depend>
+
+  <run_depend>ackermann_msgs</run_depend>
+  <run_depend>drake</run_depend>
   <run_depend>roscpp</run_depend>
   <run_depend>tf</run_depend>
-  <run_depend>drake</run_depend>
 
 
   <!-- The export tag contains other, unspecified, tags -->

